utils/docker/orakwlum-frontend/run_frontend.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+echo "Replaced Host Name with [$1]"
+server_name="$1"
+sed -i 's/$HOST_NAME/'$server_name'/g' /etc/nginx/sites-enabled/*
```

Comment:
Using the nginx conf from this repo:
https://github.com/gisce/oraKWlum-frontend/blob/master/conf/production.conf#L8

Should I use the one from oraKWlum-api-pub repo?
Then I would replace $PATH_LOG in the build script, this way the container's log will remain the same.